Manual/Classes/InstanceSynth.lean

```diff
@@ -81,6 +83,41 @@ Search may succeed, fail, or get stuck; a stuck search may occur when an unknown
 Stuck searches may be re-invoked when the elaborator has discovered one of the previously-unknown implicit arguments.
 If this does not occur, stuck searches become failures.
 
+:::example "Tracing instance search"
+
+The {option}`trace.Meta.synthInstance` option provides a powerful tool for
```

Comment:
We should also have a proper reference entry for this option. Can you add one as part of this, ideally next to these examples, so the search feature finds it directly?